apps/connect/src/components/modal/modals/settings/default-editor.tsx

```diff
@@ -15,8 +14,9 @@ const documentModelEditorOptions = [
 ] as const;
 
 export const DefaultEditor: React.FC = () => {
-    const [documentModelEditor, setDocumentModelEditor] =
-        useDefaultDocumentModelEditor();
+    const [documentModelEditor, setDocumentModelEditor] = useState<
```

Comment:
i have done this as a quick fix to silence the error, but this code should just be deleted.

we have not used the v1 document model editor in months, and as far as i am aware we are literally not even loading it in the app so selecting it here just breaks